x-pack/plugins/security/server/routes/indices/get_fields.ts

```diff
@@ -8,6 +8,20 @@ import { schema } from '@kbn/config-schema';
 import { RouteDefinitionParams } from '../index';
 import { wrapIntoCustomErrorResponse } from '../../errors';
 
+interface FieldMappingResponse {
```

Comment:
Improved type safety, at least until we can migrate away from the legacy ES client.